docs/multi-stage-query/api.md

```diff
@@ -48,7 +48,7 @@ The SQL task endpoint accepts [SQL requests in the JSON-over-HTTP form](../query
 
 This endpoint accepts [INSERT](reference.md#insert) and [REPLACE](reference.md#replace) statements.
 
-As an experimental feature, this endpoint also accepts SELECT queries. SELECT query results are collected from workers
```

Comment:
I think it is best to keep the `As an experimental feature,` language since the SELECT queries part is experimental

Reply:
agreed

Reply:
Got it